apps/backend/prisma/schema.prisma

```diff
@@ -0,0 +1,43 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id          String       @id @default(uuid())
+  username    String       @unique
+  mail        String       @unique
+  workouts    Workout[]
+  fitnessData FitnessData?
```

Comment:
I thought this one was quite weird – I wanted to model the relationship as a strict one-to-one, where neither could exist without the other. This is easier modelled than implemented, since one of the entries has to be created before the other. [You could probably do some SQL transaction thingies that ensure that a user i only created if the fitnessdata is created and so on, but we want to stick to Prisma's way of modelling and handling this.] 

I want the fitness data stored in a separate table, to make it less likely that the data is "leaked".